doc/prompt_task_map.json

```diff
@@ -0,0 +1,746 @@
+				"context-question-no-label",
+				"task_description-no-label"
+            ]
+        }
+    ],
+    "glue/qqp": [
+    ],
+    "ai2_arc/ARC-Challenge": [
+        {
+            "task": "task229_arc_answer_generation_hard",
+            "prompt": [
+				"heres_a_problem",
+				"i_am_hesitating",
+				"multiple_choice",
+				"pick_the_most_correct_option",
+				"qa_options"
+            ]
+        }
+    ],
+    "ai2_arc/ARC-Easy": [
+        {
+            "task": "task228_arc_answer_generation_easy",
+            "prompt": [
+				"heres_a_problem",
+				"i_am_hesitating",
+				"multiple_choice",
+				"pick_the_most_correct_option",
+				"qa_options"
+            ]
+        }
+    ],
+    "openbookqa/additional": [
+    ],
+    "openbookqa/main": [
+    ],
+    "piqa": [
+        {
+            "task": "task080_piqa_answer_generation",
+            "prompt": [
+				"no prompt needed"
+            ]
+        }
+    ],
+    "race/all": [
+        {
+            "task": "task309_race_answer_generation",
+            "prompt": [
+				"Select the best answer",
+				"Select the best answer (generate span)",
+				"Select the best answer (no instructions)",
+				"Taking a test"
+            ]
+        }
+    ],
+    "race/high": [
+        {
+            "task": "task309_race_answer_generation",
+            "prompt": [
+				"Select the best answer",
+				"Select the best answer (generate span)",
+				"Select the best answer (no instructions)",
+				"Taking a test"
+            ]
+        }
+    ],
+    "race/middle": [
+        {
+            "task": "task309_race_answer_generation",
+            "prompt": [
+				"Select the best answer",
+				"Select the best answer (generate span)",
+				"Select the best answer (no instructions)",
+				"Taking a test"
+            ]
+        }
+    ],
+    "hellaswag": [
+        {
+            "task": "task1389_hellaswag_completion",
+            "prompt": [
+				"Randomized prompts template",
+				"complete_first_then",
+				"how_ends",
+				"if_begins_how_continues"
+            ]
+        }
+    ],
+    "squad_v2": [
+        {
+            "task": "task349_squad2.0_answerable_unanswerable_question_classification",
+            "prompt": [
+				"Unanwerable question"
+            ]
+        }
+    ],
+    "trivia_qa/unfiltered": [
+        {
+            "task": "task1564_triviaqa_answer_generation",
+            "prompt": [
+				"first_person_context",
+				"formal_description",
+				"question_answer",
+				"question_with_instruction"
+            ]
+        }
+    ],
+    "web_questions": [
+    ],
+    "super_glue/boolq": [
+        {
+            "task": "task380_boolq_yes_no_question",
+            "prompt": [
+				"GPT-3 Style",
+				"I wonder…",
+				"after_reading",
+				"based on the following passage",
+				"based on the previous passage",
+				"could you tell me…",
+				"exam",
+				"exercise",
+				"valid_binary",
+				"yes_no_question"
+            ]
+        }
+    ],
+    "super_glue/copa": [
```

Comment:
Does `task828_copa_commonsense_cause_effect` or `task827_copa_commonsense_reasoning` correspond to this? We have been using these two tasks for our current evaluation.

Reply:
There is a slight difference between our task and their tasks. Our task is to choose the completion which is the cause or effect of the first sentence, but they have specified that the completion should be which one of them in the instances.
So, I think our task is more general and probably more difficult.

Reply:
Got it. Are we going to add a task similar to theirs?

Reply:
Yeah, I will.